### ISSUE TITLE
implement Events for Calendar

### ISSUE THREAD

**Issue by JonathanTreffler** [SchoolSquirrel/SchoolSquirrel#28]
- [ ] meetings
- [x] due dates of assignments

**Comment by hrueger** [comment]
Second part is done. However, we still need to solve the following problem:
The default due date of assignments is 23:59 which does make sense in my opinion. Many students will do work in the evening and if we do 00:00, nobody knows if that is today or tomorrow.
However, the scheduler's default visible time is from 8:00 - 18:00. So if we just display an event for a due assignment 1 hour before it is actually due, no one will see it without scrolling.

How should we go about this?

**Comment by hrueger** [comment]
Currently, the event is 12 hours long (so from 11:59 to 23:59) but this is quite a lot in the calendar.

Maybe we should change the default scheduler view from week to month view? Then, each day is a square with all events in a list.

**Comment by TessyPowder** [comment]
We could add "whole day" events wich are shown in a month style in the day view. Thats how nextcloud does it:
![image](https://user-images.githubusercontent.com/28999431/87870498-e3195b80-c9a8-11ea-8331-f638562f2daa.png)
I don't know if our calendar libary can do something like this.

**Comment by hrueger** [comment]
Yes, that is possible. It looks exactly the same like in nextcloud.
